File: scripts/Minimum Moves to Equal Array Elements II/Minimum Moves to Equal Array Elements II.cpp
class Solution {
public:
    int minMoves2(vector<int>& nums) {
        int sum=0,n=nums.size();
        sort(nums.begin(),nums.end());
        int median=nums[n/2];
        for(int i=0;i<n;i++)
            sum+=abs(nums[i]-median);     
    return sum; 
    }
};

File: algorithms/M/Minimum Moves to Equal Array Elements II/Minimum Moves to Equal Array Elements II.java
class Solution {
    public int minMoves2(int[] nums) {
        Arrays.sort(nums);
        int idx=(nums.length-1)/2;
        int sum=0;
        for(int i=0;i<nums.length;i++){
            sum+=Math.abs(nums[i]-nums[idx]);
        }
        return sum;
    }
}